package.json

```diff
@@ -138,5 +140,8 @@
   },
   "resolutions": {
     "ember-power-select/ember-concurrency": "0.8.27"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.1.1"
```

Comment:
Still not sure this is right tbh, I thought ember was all devDependencies

Reply:
ahh yeah that might have been my bad. i think you can safely move it back to dev.